src/page/Registry.js

```diff
@@ -107,25 +106,23 @@ export default function Registry() {
     contentComponent = (
       <div>
         {isDocsPage ? (
-          <p>
+          <span>
             <Link to="?">Source Code</Link>
-          </p>
+          </span>
         ) : hasDocsAvailable ? (
-          <p>
-            <Link color="primary" to="?doc">
-              Documentation
-            </Link>
-          </p>
+          <span>
+            <Link color="primary" to="?doc">Documentation</Link>
+          </span>
         ) : null}
         {state.repoUrl ? (
-          <p>
-            <Link to={state.repoUrl}>Repository</Link>
-          </p>
+          <span>
+            {hasDocsAvailable ? " | " : ""}<Link to={state.repoUrl}>Repository</Link>
+          </span>
         ) : null}
         {state.rawUrl ? (
-          <p>
-            <Link to={state.rawUrl}>Raw</Link>
-          </p>
+          <span>
+            &nbsp;|&nbsp;<Link to={state.rawUrl}>Raw</Link>
+          </span>
```

Comment:
We should try to use material-ui components rather than putting in custom text stuff like this

I'm not sure what exactly to use, but this looks appropriate: https://material-ui.com/components/dividers/#divider

See  `<Divider orientation="vertical" />`

Reply:
![d01](https://user-images.githubusercontent.com/2089210/72685003-60ddd680-3aee-11ea-8705-02af894d62db.png)

You previously agreed on pipe sep., do you want to use Material UI component instead?

Reply:
Ok @ry 
I'm using <Divider orientation="vertical"/> now with some style, please review.

Thanks